Debian #189615, #238337; Use gksu to start gdmsetup as it needs root
permissions.

Index: gdm-2.18.1/gui/gdmsetup.desktop.in.in
===================================================================
--- gdm-2.18.1.orig/gui/gdmsetup.desktop.in.in	2007-04-09 07:15:10.000000000 +0200
+++ gdm-2.18.1/gui/gdmsetup.desktop.in.in	2007-05-02 10:38:54.000000000 +0200
@@ -3,7 +3,7 @@
 _Name=Login Window
 _Comment=Configure GDM login window appearance and behavior
 TryExec=@sbindir@/gdmsetup
-Exec=@sbindir@/gdmsetup
+Exec=gksu @sbindir@/gdmsetup
 Icon=gdmsetup
 StartupNotify=true
 Terminal=false

